### DISCUSSION TITLE
Is it okay to remove the HttpApiHostMigrationsDbContext and HttpApiHostMigrationsDbContextFactory?

### DISCUSSION THREAD

**Discussion by jflaga** [abpframework/abp#17947] [Q&A]
When using the module template (`abp new MyProjectName -t module --create-solution-folder`),
... it generates an HttpApi Host module (`/host/...HttpApi.Host`) 
... with files for Migrations DbContext and Migrations DbContext Factory (`/EntityFrameworkCore/...HttpApiHostMigrationsDbContext` and `/EntityFrameworkCore/...HttpApiHostMigrationsDbContextFactory`)

Is it okay to remove these files?

**Comment by maliming** [comment]
hi

>Is it okay to remove these files?

These files has purpose, you can't remove it,
